components/studio/bin/hab-studio.sh

```diff
@@ -818,6 +818,32 @@ info() {
   return 0
 }
 
+# **Internal** Exit if current user is not root.
+ensure_root() {
+  # Early return if we are root, yay!
+  if [ $($bb id -u) -eq 0 ]; then
+    return
+  fi
+
+  # Otherwise, prepare to die with message formatting similar to the `hab` program.
+  local msg
+  local fatal
+
+  warn_msg="Running Habitat Studio requires root or administrator privileges. \
+Please retry this command as a super user or use a privilege-granting facility such as sudo."
+  fatal_msg="Insufficient permissions to complete operation"
```

Comment:
Again here I'd avoid the general "insufficient permissions" message and clearly state that we need root or sudo